ispyb-ws/src/main/java/ispyb/ws/rest/proposal/ProposalRestWebService.java

```diff
@@ -208,6 +208,32 @@ public Response updateProposal(@PathParam("token") String token, @PathParam("pro
 		}
 	}
 
+	@RolesAllowed({"User", "Manager", "Industrial", "Localcontact"})
+	@GET
+	@Path("{token}/proposal/updateProposalsByToken")
+	@Produces({ "application/json" })
+	public Response updateProposalsByToken(@PathParam("token") String token)
```

Comment:
Just for curiosity : where and when do you use this method ?

Reply:
Hi,
I use this method from EXI to update all the proposals/sessions of the loggedin user.
Alberto

Reply:
Ok, do you use it automatically at each login ? or do you provide a button do submit it ? 
I think this could be really useful for us also, because we used to have this in the old ui.

Reply:
I provide a button in the EXI user page

Reply:
Ok, Thanks! Great if you can add a form also for the manager as the old UI.